run_python_examples.sh

```diff
@@ -191,8 +195,9 @@ function stop() {
     time_sequence_prediction/traindata.pt \
     word_language_model/model.pt \
     gcn/cora/ \
-    gat/cora/ || error "couldn't clean up some files"
-
+    gat/cora/ || error "couldn't clean up some files" \
+    swin_transformer/ \
+    swin_trasformer/swin_cifar10.pt || error "command swin_transformer/swin_cifar10.pt not found" \
```

Comment:
This change does not make sense - it's wrong bash syntax. And I don't think you want to remove the whole `swin_transformer` example folder - it's managed by git. I believe you need something like this:

```suggestion
    gat/cora/ \
    swin_trasformer/swin_cifar10.pt || error "couldn't clean up some files"
```